modules/react/menu/lib/Menu.tsx

```diff
@@ -92,8 +92,8 @@ export const Menu = createContainer()({
      */
     List: MenuList,
     /**
-     * A menu item has an optional `name` prop that identifies the item in the menu list and will be
-     * passed to the optional `onSelect` callback of the menu model. A menu item can contain any
+     * A menu item has an optional `data-id` prop that identifies the item in the menu list and will
+     * be passed to the optional `onSelect` callback of the menu model. A menu item can contain any
      * HTML. If more complex HTML is provided, consider
```

Comment:
> "If more complex HTML is provided, consider..."

Consider what?

Reply:
Consider your life choices

Reply:
This section was reworked a bit. That thought was uncompleted 18 months ago. I think I meant to call out the need for text values in the collection system, but the documentation was never completed.  I've updated examples and documentation to described the purpose of text value and how to set it: https://github.com/Workday/canvas-kit/pull/2083/commits/f7d871c832668287c787226c84b465d354f2529a

Reply:
Nice to complete this thought a year and a half later! Thanks for catching this.